test/views/BaseSelectors.scala

```diff
@@ -1,24 +0,0 @@
-/*
```

Comment:
This was deleted, is there a reason why [ViewMatchers](https://github.com/hmrc/advance-valuation-rulings-frontend/blob/DDCE-4859/test/views/ViewMatchers.scala) wasn't?

Reply:
Will do as probably confusing. Left in as there are also some unused methods in the ViewSpecBase too, was unsure if we were keeping everything available to use if necessary. As these were originally copied across together